integration/ServiceTestRuleSample/app/build.gradle

```diff
@@ -17,6 +17,20 @@ android {
     }
     productFlavors {
     }
+    testOptions {
+        devices {
```

Comment:
This `devices` block has been moved to `managedDevices { devices {  } }` and will be removed soon. Could you update it?